### ISSUE TITLE
Illegal character ((CTRL-CHAR, code 27))

### ISSUE THREAD

**Issue by tinovyatkin** [sh33dafi/jest-sonar#12]
Getting following error on generated file. Looks like some characters needs to be encoded:

```sh
15:48:56.168 ERROR: Error during SonarQube Scanner execution
Error during parsing of generic test execution report '/home/runner/work/engine/engine/reports/jest-reporter.xml'. Look at the SonarQube documentation to know the expected XML format.
Caused by: com.ctc.wstx.exc.WstxUnexpectedCharException: Illegal character ((CTRL-CHAR, code 27))
 at [row,col {unknown-source}]: [4,57]
	at com.ctc.wstx.sr.StreamScanner.throwInvalidSpace(StreamScanner.java:674)
	at com.ctc.wstx.sr.StreamScanner.throwInvalidSpace(StreamScanner.java:659)
	at com.ctc.wstx.sr.BasicStreamReader.parseAttrValue(BasicStreamReader.java:1918)
	at com.ctc.wstx.sr.BasicStreamReader.handleNonNsAttrs(BasicStreamReader.java:3139)
	at com.ctc.wstx.sr.BasicStreamReader.handleStartElem(BasicStreamReader.java:2968)
	at com.ctc.wstx.sr.BasicStreamReader.nextFromTree(BasicStreamReader.java:2839)
	at com.ctc.wstx.sr.BasicStreamReader.next(BasicStreamReader.java:1073)
	at org.codehaus.staxmate.in.SMHierarchicCursor.getNext(SMHierarchicCursor.java:79)
```

Not related to #7, as testing with the fork version results in the same error.

**Comment by tinovyatkin** [comment]
https://stackoverflow.com/questions/1091945/what-characters-do-i-need-to-escape-in-xml-documents

**Comment by tinovyatkin** [comment]
That caused by ANSI characters in test name, resulting in invalid XML:
```xml
<testCase name="Line Items Creation One way transfer in [35mUSD[39m on a Minivan" duration="125" />
```

**Comment by sh33dafi** [comment]
Thanks for reporting and making a PR I will try to merge it somewhere today

**Comment by thiagotrentin** [comment]
I'm still getting this issue but when failure message have ANSI caracteres. 

In my case is something like this: 
[2mexpect([22m[31mreceived[39m[2m).[22mtoBe[2m([22m[32mexpected[39m[2m)

is it necessary to remove all te ANSI caracteres from the report?

**Comment by sh33dafi** [comment]
So if I understand you correctly @thiagotrentin you get this when a test that contains ANSI chars fails?
I will try to fix it this evening.
Thanks for reporting

**Comment by thiagotrentin** [comment]
you're right @ sh33dafi, but unlike the reported problem, in my case, the error occurs when the error message contains ANSI characters, as you can see in this example:
```
<testCase name="Testes unit&#xE1;rios para a classe CardUtils. Converte um n&#xFA;mero de cart&#xE3;o Elo e busca a imagem correspondente." duration="7">
<failure message="Error"><![CDATA[Error: [2mexpect([22m[31mreceived[39m[2m).[22mtoBe[2m([22m[32mexpected[39m[2m) // Object.is equality[22m

Expected: [32m"elo"[39m
Received: [31mnull[39m
```

**Comment by sh33dafi** [comment]
Should be fixed in v0.2.6 can you verify?

**Comment by thiagotrentin** [comment]
it works, thank you @sh33dafi !
